GameServer/Logic/Game.cs

```diff
@@ -22,17 +22,16 @@ namespace LeagueSandbox.GameServer.Core.Logic
 {
     public unsafe class Game
     {
-        ENetHost* _server;
-        BlowFish* _blowfish;
-        bool _isAlive = false;
+        private ENetHost* _server;
+        private BlowFish* _blowfish;
         private static uint dwStart = 0x40000000; //new netid
 
-        bool _started = false;
-        int playersReady = 0;
+        private bool _started = false;
+        private int playersReady = 0;
 
-        ENetPeer* currentPeer;
-        List<Pair<uint, ClientInfo>> players = new List<Pair<uint, ClientInfo>>();
-        Map map;
+        private List<Pair<uint, ClientInfo>> players = new List<Pair<uint, ClientInfo>>();
+        private Map map;
+        private System.Timers.Timer updateTimer;
```

Comment:
conventions... "Private variables in _camelCaseWithUnderscore" for all the 3 variables